apps/davi/src/stores/modules/SnapshotRepGuild/index.ts

```diff
@@ -18,8 +18,11 @@ import {
   useGetPermissions,
   useGuildConfig,
   useGetActiveProposals,
+  useGetMemberList,
 } from '../common/fetchers';
 import { useGuildConfig as useGuildConfigFromSubgraph } from '../subgraph/common/useGuildConfig';
+import { useGetMemberList as useGetMemberListFromSubgraph } from '../subgraph/common/useGetMemberList';

```

Comment:
what do you think on exporting all from an index.ts as we do for example in common/fetchers? 
so that we don't have one import line for each hook and we import them all from subgraph/common

Reply:
I agree!

Since we'll be doing a rework of the folder structure, maybe we can leave this for that PR?